warehouse/query-core/src/main/java/datawave/query/tables/BatchScannerSession.java

```diff
@@ -465,7 +466,10 @@ protected void submitScan(Scan scan, boolean increment) {
      */
     public BatchScannerSession setOptions(SessionOptions options) {
         return this;
```

Comment:
color me confused, but this method does nothing?

Reply:
That is correct. I was also confused when I saw this.